libs/portals/admin/ids-admin/src/components/Applications/Applications.tsx

```diff
@@ -1,91 +1,233 @@
+          application.applicationId.toLowerCase().includes(value.toLowerCase())
+        )
+      })
+      setApplications(filteredList)
+    } else {
+      setApplications(originalApplications)
+    }
+  }
+
+  const getHeader = ({ withCreateButton = true }) => {
+    return (
+      <GridRow rowGap={3} marginBottom={'containerGutter'}>
+        <Box
+          width={'full'}
+          display={'flex'}
+          justifyContent={'spaceBetween'}
+          columnGap={'gutter'}
+          alignItems={'center'}
+        >
+          <Stack space={2}>
             <Text variant={'h2'}>{formatMessage(m.applications)}</Text>
-          </Box>
-          <Box>
+            <Text variant={'default'}>
+              {formatMessage(m.applicationsDescription)}
+            </Text>
+          </Stack>
+          {withCreateButton && (
+            <Box>
+              <Button size={'small'}>Create Application</Button>
+            </Box>
+          )}
+        </Box>
+      </GridRow>
+    )
+  }
+
+  return applications.length === 0 ? (
+    <GridContainer>
+      {getHeader({ withCreateButton: false })}
+      <GridRow>
+        <Box
+          display={'flex'}
+          flexDirection={'column'}
+          border={'standard'}
+          borderRadius={'large'}
+          justifyContent={'center'}
+          alignItems={'center'}
+          padding={10}
+        >
+          <Text variant={'h3'}>No application created</Text>
+          <Text paddingTop={'gutter'}>
+            Lorem ipsum dolor sit amet consectetur. A non ut nulla vitae mauris
+            accumsan at tellus facilisi.
+          </Text>
+          <Box marginTop={6}>
             <Button size={'small'}>Create Application</Button>
           </Box>
+          <Box marginTop={'gutter'}>
+            <Button variant={'text'}>Learn more</Button>
+          </Box>
         </Box>
       </GridRow>
+    </GridContainer>
+  ) : (
+    <GridContainer className={styles.relative}>
+      {getHeader({})}
       <Box paddingTop={'gutter'}>
         <Stack space={[1, 1, 2, 2]}>
-          {MockApplications.map((item) => (
-            <GridRow key={item.id}>
+          <GridRow>
+            <Filter
+              variant={'popover'}
+              align="left"
+              reverse
+              labelClear={formatMessage(m.clearFilter)}
+              labelClearAll={formatMessage(m.clearAllFilters)}
+              labelOpen={formatMessage(m.openFilter)}
+              labelClose={formatMessage(m.closeFilter)}
+              resultCount={0}
+              filterInput={
+                <FilterInput
+                  placeholder={formatMessage(m.searchPlaceholder)}
+                  name="session-nationalId-input"
+                  value={inputSearchValue}
+                  onChange={handleSearch}
+                  backgroundColor="blue"
+                />
+              }
+              onFilterClear={() => {
+                setInputSearchValue('')
+              }}
+            ></Filter>
```

Comment:
~you should close like `/>` instead of `></Filter>`

Reply:
We will add content in there for the filter dropdown, so I will leave it as is :D